src/main/twirl/scalajs/jsdeps.scala.html

```diff
@@ -1,3 +1,5 @@
+@import play.twirl.api.TwirlFeatureImports.defining
```

Comment:
I do not see any warnings when compiling the code. What compiler options & Scala version are you using?